packages/eslint-plugin-pf-codemods/src/rules/v6/chipReplaceWithLabel/chipReplaceWithLabelInput.tsx

```diff
@@ -0,0 +1,3 @@
+import { Chip } from '@patternfly/react-core/deprecated';
+
+export const ChipReplaceWithLabelInput = () => <Chip onClick=(handleClick) badge={badge}></Chip>;
```

Comment:
I think this was meant to be `onClick={handleClick}`.
Also can we add some text as children of the Chip? (to better demonstrate the badge transformation)